languagetool-language-modules/zh/src/main/resources/org/languagetool/rules/zh/zh-CN/grammar.xml

```diff
@@ -0,0 +1,7314 @@
+                </message>
+                <example correction=""><marker>抵砺</marker>革命意志。</example>
+                <example>砥砺革命意志。</example>
+            </rule>
+        </rulegroup>
+        <rulegroup id="SHA1_SHA2" name="霎/刹">
+            <rule>
+                <pattern>
+                    <marker>
+                        <token>霎车</token>
+                    </marker>
+                </pattern>
+                <message>
+                    您的意思是"刹车"吗？
+                </message>
+                <example correction="">这是<marker>霎车</marker>的痕迹吗？</example>
+                <example>这是刹车的痕迹吗？</example>
+            </rule>
+        </rulegroup>
+        <rulegroup id="SHI1_SHI2" name="是/试">
+            <rule>
+                <pattern>
+                    <marker>
+                        <token>试</token>
+                    </marker>
+                </pattern>
+                <message>
+                    您的意思是"是"吗？
+                </message>
+                <example correction="">我<marker>试</marker>中国人。</example>
+                <example>我是中国人。</example>
+            </rule>
+            <rule>
+                <pattern>
+                    <marker>
+                        <token>试</token>
+                    </marker>
+                </pattern>
+                <message>
+                    您的意思是"是"吗？
+                </message>
+                <example correction="">我<marker>试</marker>可爱的中国人。</example>
+                <example>我是可爱的中国人。</example>
+            </rule>
+        </rulegroup>
+        <rulegroup id="TU1_TU2" name="突/凸">
+            <rule>
+                <pattern>
+                    <marker>
+                        <token>凸然</token>
+                    </marker>
+                </pattern>
+                <message>
+                    您的意思是"突然"吗？
+                </message>
+                <example correction="">这事情发生的太<marker>凸然</marker>了。</example>
+                <example>这事情发生的太突然了。</example>
+            </rule>
+            <rule>
+                <pattern>
+                    <marker>
+                        <token>凸变</token>
+                    </marker>
+                </pattern>
+                <message>
+                    您的意思是"突变"吗？
+                </message>
+                <example correction="">它的<marker>凸变</marker>让我很害怕。</example>
+                <example>它的突变让我很害怕。</example>
+            </rule>
+        </rulegroup>
+        <rulegroup id="TUO1_TUO2" name="拖/托">
+            <rule>
+                <pattern>
+                    <marker>
+                        <token>拖运</token>
+                    </marker>
+                </pattern>
+                <message>
+                    您的意思是"托运"吗？
+                </message>
+                <example correction="">我要<marker>拖运</marker>包裹。</example>
+                <example>我要托运包裹。</example>
+            </rule>
+            <rule>
+                <pattern>
+                    <ma
```

Comment:
I see there are many rules without a `<suggestion></suggestion>` and thus with empty `correction` attributes. Is this on purpose? Is there no way to offer a suggestion for these rules?

Reply:
In the old version which maintained by Tao Lin, lots of rules are formed like you mentioned. So I keep it, but I think part of them should actually have a `<suggestion></suggestion>`. So should I fix it?

Reply:
Yes, please add  a suggestion where it makes sense.